app/scripts/filters/resources.js

```diff
@@ -1351,8 +1316,21 @@ angular.module('openshiftConsole')
     };
   })
   .filter('hasAlternateBackends', function() {
-    return function(route) {
+    return function (route) {
       var alternateBackends = _.get(route, 'spec.alternateBackends', []);
       return !_.isEmpty(alternateBackends);
     };
+  })
+  // TODO: these two filters are defined in origin-web-catalog but are not available, need to figure out why or move to common
+  .filter('applicationHasDeployment', function ($filter) {
```

Comment:
We have both filters already in origin-web-console

https://github.com/openshift/origin-web-console/blob/master/app/scripts/filters/resources.js#L719-L728

Although if we move them to common it might be better to remove the dependency on DeploymentsService so we don't have to move the entire service.